src/main/java/com/mcmoddev/mmdbot/modules/commands/bot/info/CmdUptime.java

```diff
@@ -39,9 +40,8 @@ protected void execute(final CommandEvent event) {
 
         embed.setTitle("Time spent online.");
         embed.setColor(Color.GREEN);
-        //TODO Fix utils that handle time to show hours and minuets too.
-        embed.addField("I've been online for: ", Utils.getTimeDifference(Utils.getLocalTime(
-            References.STARTUP_TIME), LocalDateTime.now()), false);
+        embed.addField("I've been online for: ", Utils.getTimeDifference(Utils.getTimeFromUTC(
+            References.STARTUP_TIME), OffsetDateTime.now(ZoneOffset.UTC)), false);
```

Comment:
Hmm, this gives me the thought that the bot uptime command should probably include minutes and seconds too in its output.

Reply:
This would require multiple versions of `getTimeDifference` that includes minutes and seconds, lest it be extremely verbose for every printout.

Reply:
Well, we could change this call to use the overload which accepts the `ChronoUnit`s directly, and pass it the units for months to seconds.